tests/imap_test.in

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I think I mentioned it in the previous PR, but `imap` is very misleading, and makes me think of Internet Message Access Protocol (especially since we are sending email). Maybe `img_map` or something?